core/rfb.js

```diff
@@ -424,7 +425,11 @@ export default class RFB extends EventTargetMixin {
         // Make our elements part of the page
         this._target.appendChild(this._screen);
 
+        // We don't want to reuse the cursor from the old connection
+        // but maybe want to show a dot instead
```

Comment:
I don't understand this comment. RFB objects only exist for a single connection.

Reply:
OK, so I misunderstood the logic, the comment is wrong, and the code is suboptimal, because the cursor is initialized twice. Will rework.